pkg/scheduler/framework/runtime/metrics_recorder.go

```diff
@@ -43,18 +43,18 @@ type metricsRecorder struct {
 
 	// stopCh is used to stop the goroutine which periodically flushes metrics. It's currently only
```

Comment:
the comment is now inaccurate. @kerthcet could you fix that?

Reply:
addressed here https://github.com/kubernetes/kubernetes/pull/110297